tests/testthat/test-fbref.R

```diff
@@ -254,7 +254,9 @@ test_that("fb_team_match_results() works", {
 #   # expect_false(nrow(scout) == 0)
 #
 #   scout_secondary <- fb_player_scouting_report(player_url = "https://fbref.com/en/players/d70ce98e/Lionel-Messi",
-#                                      pos_versus = "secondary", time_pause = 4)
```

Comment:
Have now commented this test out again. Locally it passes, but when tested on GHA it doesn't seem to like the `.load_page()` method of loading html. Now I remember this was always the issue with this particular function for some strange reason...